install/threesdk/InstallTools.py

```diff
@@ -1873,19 +1873,41 @@ def _installbase_for_shell():
             Tools.execute(script, interactive=True, sudo_remove=sudoremove)
 
     @staticmethod
-    def get_newest_version():
-        from threesdk import __version__
+    def is_latest_release():
+        try:
```

Comment:
why don't you just reuse get_latest_release here?

Reply:
reused, thanks Jo :+1: